src/modules/pages/components/RouteLayouts/UserNavigation.css

```diff
@@ -168,3 +168,20 @@
   border-radius: var(--radius-normal);
   background-color: color-mod(var(--temp-grey-blue-7) alpha(10%));
 }
+
+.walletAutoLogin {
+  margin-right: 12px;
+  padding: 3px 10px 0;
+  height: 24px;
```

Comment:
The user reputation on the left to the `connecting wallet` part is 26px - 2 px more than walletAutoLogin (and it does look a bit bigger as per the picture).
Is it intentional? I didn't find the figma specs to check.

<img width="436" alt="Screenshot 2021-07-12 at 11 58 57" src="https://user-images.githubusercontent.com/34057551/125260779-9422af80-e2f8-11eb-8807-459c5a68ea26.png">

Reply:
_Ahahahaha specs!_ There's no specs :)

But no, it's not intentional, I just did what looked good to me :P

But I'll happily increase it to `26` so that it's the same height